.gitignore

```diff
@@ -1,4 +1,3 @@
 node_modules
 .idea
 build
-coverage
```

Comment:
Not sure this is the correct way of including coverage. I believe what you have to provide is the coverage result file, not all the folder. So probably, we should ignore everything except the file. We can test alternatives until checking what works.

Reply:
You are right, I excluded just the `.info` file